### DISCUSSION TITLE
special german characters

### DISCUSSION THREAD

**Discussion by skalec** [netbox-community/netbox#7895] [General]
Hello community,

can someone help me and tell me what to set so that I can use German sionder characters. Ä Ü Ö.

i think it's not a netbox but a system problem. I use an Ubuntu system

**Comment by amk1969** [comment]
Hi skalec,

My Ubuntu system has no issues using any Unicode character, but you will have to be more specific about the issues you encounter. And also about the background of raising this question in a Netbox forum while admitting that it is not a Netbox issue.

With the limited information about your problem it is not possible to help directly, but this link can help to make your request more useful and also to direct it to a more appropriate place: http://www.catb.org/~esr/faqs/smart-questions.html#intro
